.github/workflows/push.yml

```diff
@@ -41,10 +43,10 @@ jobs:
 
           cat << EOF > test.mjs
             import { createLinter } from 'actionlint';
-            createLinter()('on: psuh', 'push.yml').then(
-              (results) => process.exit(results.length > 0 ? 0 : 1),
-              (err) => { console.error(err); process.exit(1); }
-            );
+            createLinter().then(lint => {
+              const results = lint('on: psuh', 'push.yml');
```

Comment:
`on: psuh` should probably be `on: push`, right?
Is there a test where this could fail?

Reply:
No, this is an integration test and we want to see an error.

Reply:
Nevermind :smiley: